pkg/service/test/test.go

```diff
@@ -209,7 +195,14 @@ func (t *tester) Test(path string, testReportPath string, appCmd string, options
 		MongoPassword:      options.MongoPassword,
 		WithCoverage:       options.WithCoverage,
 		CoverageReportPath: options.CoverageReportPath,
-		EnableTele:         enableTele,
+		Tele:               tele,
+		TestReport:         testReportStorage,
+		Storage:            tcsStorage,
+	}
+	sessions, err := cfg.TestReport.ReadSessionIndices(path, t.logger)
```

Comment:
isn't sessions must be related to testcases why are we getting sessions from testReport..?